lms/static/js/fixtures/dashboard/dashboard.html

```diff
@@ -31,8 +31,8 @@ <h3 class="course-title">
     <span class="xseries-icon" aria-hidden="true"></span>
     <p class="message-copy">XSeries Program Course</p>
   </div>
-    <article class="course honor">
-        <section class="details">
+    <div class="course honor">
+        <div class="details" aria-labelledby="course-details-heading">
```

Comment:
Chris, I know this is just a fixture file, but there is no element in here with ID course-details-heading.